thirdparty/nginx/nginx-1.7.1-zimbra/src/mail/ngx_mail_throttle_module.h

```diff
@@ -22,22 +22,23 @@
 #include <ngx_memcache.h>
 
 struct ngx_mail_throttle_srv_conf_s {
-    ngx_uint_t  mail_login_ip_max;
-    ngx_msec_t  mail_login_ip_ttl;
-    ngx_str_t   mail_login_ip_ttl_text;
-    ngx_str_t   mail_login_ip_imap_ttl_text;
-    ngx_str_t   mail_login_ip_pop3_ttl_text;
-    ngx_str_t   mail_login_ip_rejectmsg;
-    ngx_uint_t  mail_login_user_max;
-    ngx_msec_t  mail_login_user_ttl;
-    ngx_str_t   mail_login_user_ttl_text;
-    ngx_str_t   mail_login_user_rejectmsg;
-    ngx_uint_t  mail_login_ip_imap_max;
-    ngx_msec_t  mail_login_ip_imap_ttl;
-    ngx_uint_t  mail_login_ip_pop3_max;
-    ngx_msec_t  mail_login_ip_pop3_ttl;
+    ngx_uint_t   mail_login_ip_max;
+    ngx_msec_t   mail_login_ip_ttl;
+    ngx_str_t    mail_login_ip_ttl_text;
+    ngx_str_t    mail_login_ip_imap_ttl_text;
+    ngx_str_t    mail_login_ip_pop3_ttl_text;
+    ngx_str_t    mail_login_ip_rejectmsg;
+    ngx_uint_t   mail_login_user_max;
+    ngx_msec_t   mail_login_user_ttl;
+    ngx_str_t    mail_login_user_ttl_text;
+    ngx_str_t    mail_login_user_rejectmsg;
+    ngx_uint_t   mail_login_ip_imap_max;
+    ngx_msec_t   mail_login_ip_imap_ttl;
+    ngx_uint_t   mail_login_ip_pop3_max;
+    ngx_msec_t   mail_login_ip_pop3_ttl;
```

Comment:
Yep I wanted to make it line up using the same conventions that I noticed being followed elsewhere in the code.  When I added the `mail_throttle_whitelist_ips`, its type `ngx_array_t` was long enough to make the others not line up correctly.  

So, pointer vars (like `*mail_throttle_whitelist_ips`) are set to the nearest 4-space indent level and the non-pointer vars (like `mail_login_ip_pop3_ttl`) are indented an extra space.